Source/WebInspectorUI/UserInterface/Views/SingleSidebar.css

```diff
@@ -47,6 +47,16 @@
     border-inline-end: 1px solid var(--border-color);
 }
 
-.single-sidebar.trailing {
+body:not(.narrow) .single-sidebar.trailing {
     border-inline-start: 1px solid var(--border-color);
 }
+
+body.narrow .sidebar.trailing .single-sidebar {
+    width: 100% !important;
+    border-top: none;
+    min-height: auto;
```

Comment:
is this actually necessary?

Reply:
It probably was at some point in the past but doesn't seem to be anymore.